src/utils/helpers/userRole.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable import/prefer-default-export */
+import models from '../../database/models';
+
+const { Users } = models;
+const getUserRole = async (id) => {
+  const user = await Users.findOne({ where: { id: `${id}` } });
+  return user.roleId;
```

Comment:
will fail if role ID isn't found. Also is the type or roleId string or number? Why use string literals?